### ISSUE TITLE
An issue with loading YOLOv5s

### ISSUE THREAD

**Issue by SoroushMaleki** [ultralytics/yolov5#7662]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### YOLOv5 Component

PyTorch Hub

### Bug

Hello,

I have run into an issue when loading a custom-trained YOLO v5 structure. The issue is that after I set the "force_load" parameter to "True" using the following line when loading the model:
`torch.hub.load('ultralytics/yolov5', 'custom', "/home/soroush/nodelab/src/vision_node/src/best.pt", force_load=True )`

The torch.hub is no longer able to load the YOLO v5 model and gets stuck when "Adding Autoshape". Here is how it looks like in the command line:

`Using cache found in /home/soroush/.cache/torch/hub/ultralytics_yolov5_master
YOLOv5 🚀 2022-5-1 torch 1.11.0 CUDA:0 (NVIDIA GeForce GTX 1050, 4042MiB)

Fusing layers... 
YOLOv5s summary: 213 layers, 7018216 parameters, 0 gradients
Adding AutoShape...`

And nothing else happens ...
Can anyone explain how to solve this issue?

Thanks.

### Environment

YOLOv5 🚀 2022-5-1 torch 1.11.0 CUDA:0 (NVIDIA GeForce GTX 1050, 4042MiB)
OS: Ubuntu 20.0

### Minimal Reproducible Example

```
weight_path = "fake_path/best.pt"
torch.hub.load('ultralytics/yolov5', 'custom', path=weight_path, force_reload=True)
```

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@SoroushMaleki 👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. PyTorch Hub inference with custom models works correctly in our test:

<img width="1049" alt="Screen Shot 2022-05-01 at 3 43 48 PM" src="https://user-images.githubusercontent.com/26833433/166167558-1d2ea0f2-6a6a-4ac6-b330-fc277778a9af.png">

We've created a few short guidelines below to help users provide what we need in order to start investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and diagnose your problem. 

Thank you! 😃
